Core/test/qa-functional/src/org/sleuthkit/autopsy/ingest/IngestFileFiltersTest.java

```diff
@@ -156,6 +156,34 @@ public void testBasicDir() {
         }
     }
     
+    public void testExtAndDirWithOneRule() {
+        HashMap<String, Rule> rules = new HashMap<>();
+        rules.put("Rule", new Rule("testExtAndDirWithOneRule", new Rule.ExtensionCondition("jpg"), new MetaTypeCondition(MetaTypeCondition.Type.FILES), new ParentPathCondition("dir1"), null, null, null));
+        //Build the filter that ignore unallocated space and with one rule
+        FilesSet Files_Ext_Dirs_Filter = new FilesSet("Filter", "Filter to find all jpg files in dir1.", false, true, rules);
+        
+        try {
+            Case openCase = Case.getOpenCase();
+            runIngestJob(openCase.getDataSources(), Files_Ext_Dirs_Filter); 
+            FileManager fileManager = Case.getOpenCase().getServices().getFileManager();            
+            List<AbstractFile> results = fileManager.findFiles("%%");
+            assertEquals(62, results.size());
```

Comment:
I'm not sure of the purpose of this assert. DO you have doubts about whether or not FileManager.findFiles is working?

Reply:
To me this seems like a reasonable check. While I wouldn't expect findFiles to fail, if it did and returned, say, zero files, then the tests we want to run wouldn't execute at all and also wouldn't generate any warnings (since we looped over zero files).

Reply:
Excellent point, @APriestman, thanks!